oak_functions/proto/abi.proto

```diff
@@ -21,6 +21,20 @@ package oak.functions.abi;
 option java_multiple_files = true;
 option java_package = "oak.functions.abi";
 
+// ChannelStatus returned from a read, write, or wait on a channel.
+enum ChannelStatus {
```

Comment:
Isn't this more like an operation status, and the one below a channel status? That is definitely not a channel handle.

Reply:
I agree---ChannelStatus is more like an operation status. Or more precise: the status code after an operation on a channel: How about ChannelStatusCode?

Which one below do you mean: the `ChannelHandle`? Do you think that one is wrongly named?

Reply:
Yes, `ChannelHandle` is the "numbering"/extension of the channel---so the first argument of `channel_read/write/wait()` in the ABI [1]. I'll add some documentation! 

I am worried that `ChannelStatus` -> `Status` looses too much information and I used `ChannelStatus` to distinguish the Status after a (low-level) channel operation from the (high-level SDK) `OakStatus`. I am trying to come up with a better proposal. 

[1]  https://github.com/project-oak/oak/blob/e293c803f6c7d373fb963aac86841934699f7982/oak_functions/abi/src/lib.rs#L108

Reply:
Will there be a difference between low level and high level status eventually?

Reply:
I see the difference in: the "low level" gives a status concerning the channel operations and is independent from Oak. The "high level" status is not dependent on Oak (and the Channel Handles)